src/Error/ExceptionRenderer.php

```diff
@@ -19,7 +19,6 @@
 use Cake\Core\Configure;
 use Cake\Core\Exception\Exception as CakeException;
 use Cake\Core\Exception\MissingPluginException;
-use Cake\Error\PHP7ErrorException;
```

Comment:
@ADmad  Yeah but the file namespace is ``Cake\Error`` so you can use it directly without import.